addon/services/building.js

```diff
@@ -95,6 +96,56 @@ export default class BuildingService extends GwrService {
     return this.createAndCache(xml);
   }
 
+  async createAndAddToConstructionProject(EPROID, buildingWork) {
+    let body = this.xml.buildXMLRequest("addWorkToProject", buildingWork);
+    let response = await this.authFetch.fetch(
+      `/constructionprojects/${EPROID}/work`,
```

Comment:
what is the difference between this and the call in `.create`?

Reply:
It allows buildings to be added that are not in the state existing or unusable. This is all the information I found:
![image](https://user-images.githubusercontent.com/32454507/129362341-25c8b50c-9a2d-45fd-9895-280ad2d79878.png)